site/content/entry/citra-progress-report-2020-q2.md

```diff
@@ -0,0 +1,699 @@
+
+It required quite a bit of work to get the cores in sync while not ruining performance. Developer [B3n30](https://github.com/B3n30) researched the scheduler and stepped up to implement what we call "Core timing 2.0". He added a timer for each core, and basically rewrote how our timing worked, thus creating a more accurate scheduler. After this change, not only was a big step done towards compatibility with New 3DS games, but it also fixed all the games that were broken with the priority boost hack removed, for instance, [Game].
+
+While Core timing 2.0 inproved timing accuracy, it also came with performance hits, such as the infamous stutter when entering battles in Pokemon games. One reason for such regressions was that, since there are more cores to take care of, cache invalidation would happen more often and  on more cores. [MerryMage](https://github.com/MerryMage), author of [dynarmic](https://github.com/MerryMage/dynarmic), the ARM JIT we are using, fixed the Pokemon stutters by reducing overhead of such invalidations. However, some other games still had problems. [B3n30](https://github.com/B3n30) found that this was due to the JIT executing smaller slices, and he is now working to strike a proper balance between accuracy and performance. Some of his changes are already in Canary, so be sure to check them out!
+
+### core: Add support for N3DS memory mappings ([#5103](https://github.com/citra-emu/citra/pull/5103)) by [FearlessTobi](https://github.com/FearlessTobi)
+
+3DS games use "memory modes" to tell the kernel how much memory they would use, and how much memory is left for the Home Menu, applets and services. As mentioned above, the New 3DS has about double the amount of RAM, so there are also more memory modes we need to support. With the help of other team members, [FearlessTobi](https://github.com/FearlessTobi) managed to make an accurate implementation, making it possible for New 3DS games to map the extra RAM as they like.
+
+### CFG: Let GetSystemModel report model based on Settings:is_new_3ds ([#5104](https://github.com/citra-emu/citra/pull/5104)) by [B3n30](https://github.com/B3n30)
+
+After those two changes were thoroughly tested and eventually merged, all we had to do was to inform the games they were running in New 3DS mode. For the most part this had already been done even before we actually implemented New 3DS support, but there were still some edge cases left unhandled.
+
+One of these was the CFG service. Games are able to get from this service lots of information about the console they are running on. As one'd expect, this information also included the model of the system. [B3n30](https://github.com/B3n30) updated the function `GetSystemModel` so that it would change the return value to New 3DS if Citra is set to New 3DS mode.
```

Comment:
I think both version are grammatically correct, is there a specific reason for these changes?

Reply:
> 
> 
> I think both version are grammatically correct, is there a specific reason for these changes?

I changed the first one because of grammar and the second one because it sounded more former in my opinion.